Changelog.md

```diff
@@ -1,5 +1,9 @@
 # Fossa CLI Changelog
 
+## v3.1.5
```

Comment:
Aren't we supposed to create an "Unreleased" section?  This saves a step of changing it when doing a release, but doesn't follow the process in our docs.  This feels simpler, but gives the false impression that this release exists.

Reply:
We do this when we're planning to release immediately.